data/json/items/containers.json

```diff
@@ -112,7 +112,10 @@
         "to_hit":-1,
         "material":"steel",
         "contains":1,
-        "rigid": true
+        "rigid": true,
+        "watertight": true,
+        "seals": true,
+        "qualities": [["CONTAIN", 1]]
```

Comment:
If you look at the recipe for making the clay bowls, they have waterproofing materials and a fur or leather lid, the petwer bowl doesn't have a lid.